### PR TITLE
[12.0][REF] stock_demand_estimate: split into 2 modules

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Split into:

* stock_demand_estimate: just the datamodel.
* stock_demand_estimate_matrix (any suggestion for a better name?): adds the wizard to assist on the estimates creation and update.

@ForgeFlow

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]

File: stock_demand_estimate_matrix/models/stock_demand_estimate.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    @api.multi
+    @api.depends(
+        "date_range_id", "manual_duration", "manual_date_from", "manual_date_to",
+    )
+    def _compute_dates(self):
+        date_range_records = self.filtered(lambda r: r.date_range_id)
+        res = super(
+            StockDemandEstimate, self - date_range_records)._compute_dates()
+        for rec in date_range_records:
+            rec.date_from = rec.date_range_id.date_start
```
Comment (HviorForgeFlow) [inline stock_demand_estimate_matrix/models/stock_demand_estimate.py:25]: Instead of multiple assignation a write could be used

Comment (LoisRForgeFlow) [reply stock_demand_estimate_matrix/models/stock_demand_estimate.py:25]: This is a compute method, you cannot do that

Comment (LoisRForgeFlow) [reply stock_demand_estimate_matrix/models/stock_demand_estimate.py:25]: plus it depends on the date_range_id of each record

**Review by AdriaGForgeFlow — APPROVED** [review]
Functional review :+1:

**Pull request merged**